composer.json

```diff
@@ -11,13 +11,14 @@
     "require": {
         "php": "^7.1.0",
         "ext-mbstring": "*",
-        "nette/utils": "2.5.*",
-        "nette/di": "2.3.*|2.4.*",
-        "nette/neon": "2.*"
+        "nette/utils": "~2.5.4",
+        "nette/di": "~2.3.13|~2.4.14",
```

Comment:
Je tu ještě nutnost ponechávat 2.3 verzi?

Reply:
Pokud někdo používá Nette/DI 2.3 (a myslím, že naše projekty ještě ano), tak je to fajn mít zároveň podporované, protože v případě fixu pro starší verzi nemusí vznikat druhá větev. Tento projekt to nijak funkčně neovlivňuje, ani nelimituje.

Reply:
Navíc je možné používat současně stejnou verzi při přechodu na Nette 2.4 a nemění se najedou takové množství knihoven.

Reply:
Asi bych na to kaslal. Povysim setinkovou verzi a dropnu podporu pro `2.3`. Pokud bysme potrebovali fixovat, vydame bugfix do verze `1.0.x`